api/src/main/java/org/openmrs/module/kenyaemr/reporting/library/shared/common/CommonCohortLibrary.java

```diff
@@ -161,7 +155,7 @@ public CohortDefinition hasObs(Concept question, Concept... answers) {
 		cd.setName("has obs between dates");
 		cd.setQuestion(question);
 		cd.setOperator(SetComparator.IN);
-		cd.setTimeModifier(PatientSetService.TimeModifier.ANY);
+		//cd.setTimeModifier(PatientSetService.TimeModifier.ANY);
```

Comment:
why are we modifying this? This may have effects on code places it has currently been used